### DISCUSSION TITLE
Playback error on Android devices

### DISCUSSION THREAD

**Discussion by flyinfalcon** [blakeblackshear/frigate#12667] [General Support]
While playing event clips on Android devices(Samsung) I frequently get this error.

![image](https://github.com/user-attachments/assets/87bab0f5-a3b8-4ebb-aab7-3dc329c1a83c)

This occurs on all popular browsers:chrome, Firefox etc...

The cameras are h264 streams and am using a nginx reverse proxy to access frigate 

I have tried reducing resolution, using webview beta but no luck.

This does not happen on my iPhone or PC, any ideas what causes this?

Thanks

**Comment by ElectronicBattle** [comment]
Interestingly, I have the opposite problem! I can play event clips in/on my Android tablets and phones  but not in my browsers on linux.

these two threads of mine might be of use:
https://github.com/blakeblackshear/frigate/discussions/12406
https://github.com/blakeblackshear/frigate/discussions/12647

In my case, as far as I have been able to tell (in amongst my failing to grasp how to configure the go2rtc and other parts of the config) it is to do with some browsers (most) being unable to render h265.

I *have* eventually got it to work in linux but at the expense of huge CPU demands and my GPU seemingly unutilised[*] in spite of settings suggesting the contrary, so I have had to fall back to the situation where my CPU is hardly taxed, clips play if i save them and use a media player (ffplay, mplayer, vlc) but  in-browser playback then is not possible.  I don't know if anyone knows how to solve my problem but perhaps you'll get some ideas from mine (I am out of my depth; I am just a hobby user and I don't work in IT).

Please let me know how you get on - I too would like to solve this matter!

Also - there's an app for Android called [Frigate Viewer](https://play.google.com/store/apps/details?id=com.frigateviewer) which might perhaps help determine what's happening.

[*_edit - I have since discovered that my GT1030 supports ndec decoding but not nvenc encoding, so transcoding is not an option for me_]

EB
